### PR TITLE
Configurable projector header

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/core/config_variables.py [hunk 1]
```diff
@@ -139,8 +139,8 @@ def get_config_variables():
         default_value=True,
         input_type='boolean',
         label='Show logo on projector',
-        help_text='You can replace the logo. Just copy a file to '
-                  '"static/img/logo-projector.png" in your OpenSlides data path.',
+        help_text='You can replace the logo by uploading an image and set it as '
+                  'the "main projector logo" in the mediafiles.',
```
Comment (emanuelschuetze) [inline openslides/core/config_variables.py]: We use "files" instead of "mediafiles" in UI. 
Suggestion: ' .. in "files".'

Comment (FinnStutzenstein) [inline openslides/core/config_variables.py]: Done.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Ping.

**Comment by FinnStutzenstein** [comment]
Rebased.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Whats about moving this option into the projector manage view, so you can enable/disable the clock for each projector on its own?

**Comment by emanuelschuetze** [comment]
@normanjaeckel I think the new clock config fits well in the projector config because there we manage also other projector appearance (e.g. show logo/text, color etc). All configs has effect to _all_ projectors. So I am not sure if this is a good idea to move only the clock to the manage projector view.

+1  for merging this pull request without any changes and think about later about more ui improvements.
Ok?

**Pull request merged**